cypher/cypher-docs/src/docs/dev/glossary.asciidoc

```diff
@@ -270,3 +272,70 @@ This section comprises a glossary of all the keywords -- grouped by category and
+| <<administration-security-administration-database-tokens, CREATE NEW RELATIONSHIP TYPE>>      | Schema                | Determines whether a user is allowed to create new relationship types on a specific database.
+| <<administration-security-administration-database-tokens, CREATE NEW PROPERTY NAME>>          | Schema                | Determines whether a user is allowed to create new property names on a specific database.
+| <<administration-security-administration-database-tokens, NAME MANAGEMENT>>                   | Schema                | Determines whether a user is allowed to create new labels, types and property names on a specific database.
+| <<administration-security-administration-database-all, ALL DATABASE PRIVILEGES>>              | Database and schema   | Implies all privileges from the categories Database and Schema for a specific database.
+| <<administration-security-subgraph-traverse, TRAVERSE>>                                       | GRAPH                 | Determines whether an element (node, relationship or both) can be found and traversed while executing queries on the specified graph.
+| <<administration-security-subgraph-read, READ>>                                               | GRAPH                 | Determines whether the properties of an element (node, relationship or both) can be read while executing queries on the specified graph.
+| <<administration-security-subgraph-match, MATCH>>                                             | GRAPH                 | Determines whether the properties of an element (node, relationship or both) can be read and the element can be found and traversed while executing queries on the specified graph.
+| <<administration-security-subgraph-write, WRITE>>                                             | GRAPH                 | Determines whether the user can execute write operations on the specified graph.
+| <<administration-security-administration-dbms-privileges-role-management, CREATE ROLE>>       | DBMS                  | Determines whether the user can create new roles.
+| <<administration-security-administration-dbms-privileges-role-management, DROP ROLE>>         | DBMS                  | Determines whether the user can delete roles.
+| <<administration-security-administration-dbms-privileges-role-management, ASSIGN ROLE>>       | DBMS                  | Determines whether the user can grant roles.
+| <<administration-security-administration-dbms-privileges-role-management, REMOVE ROLE>>       | DBMS                  | Determines whether the user can revoke roles.
+| <<administration-security-administration-dbms-privileges-role-management, SHOW ROLE>>         | DBMS                  | Determines whether the user can get information about existing and assigned roles.
+| <<administration-security-administration-dbms-privileges-role-management, ROLE MANAGEMENT>>   | DBMS                  | Determines whether the user can create, drop, grant, revoke and show roles.
```

Comment:
Should this list `assign` and `remove` roles instead of `grant` and `revoke` since that is what the other privileges are called?

Reply:
The way I structured this table is that I separated the name of the privilege from the "action" that can be done with it in the description. So as a user you can see that having "ASSIGN ROLE" means that you can "grant" (like the command!). That's why I would like to keep it this way